examples/oil-gas-automation/.env.example

```diff
@@ -1,8 +1,3 @@
-# OpenAI's API key to access model completions
 VITE_OPENAI_API_KEY=
-
-# URL under which backend endpoints are accessible
 VITE_API_URL=
-
-# URL under which backend subscription is accessible
 VITE_API_WS_URL=
```

Comment:
Are we sure those explanatory comments should be removed?

Reply:
There should be no changes to this example in the PR, fixing that